libcxx/test/std/containers/associative/multimap/scary.pass.cpp

```diff
@@ -8,20 +8,19 @@
 
 // <map>
 
-// class map class multimap
+// class map
+// class multimap
 
-// Extension:  SCARY/N2913 iterator compatibility between map and multimap
+// Extension: SCARY/N2913 iterator compatibility between map and multimap
 
 #include <map>
 
 #include "test_macros.h"
 
-int main(int, char**) {
+void test() {
```

Comment:
I think these files should be renamed to `*.compile.pass.cpp` as the `main` functions are removed, and, IIUC, runtime detection would be redundant if iterator types are the same.